gpkit/constraints/prog_factories.py

```diff
@@ -5,7 +5,7 @@
 try:
     from ipyparallel import Client
     CLIENT = Client(timeout=0.01)
-    assert len(CLIENT) > 0
+    assert len(CLIENT) > 0  # pylint:disable=len-as-condition
```

Comment:
I think the more pythonic thing is for empty iterables to implement `__bool__` to return False, but I wasn't sure if `Client` implements that behavior.